### DISCUSSION TITLE
TIL - 22/03/16

### DISCUSSION THREAD

**Discussion by leeheefull** [selab-hs/selab#131] [Check TIL]
@leeheefull @DongGeon0908

**Comment by leeheefull** [comment]
### 작업 사항
* ["Transaction을 주의하여 Async 사용하기"](https://github.com/selab-hs/selab-blog-api/wiki/%5B%ED%9D%AC%EC%B0%AC%5D-Transaction%EC%9D%84-%EC%A3%BC%EC%9D%98%ED%95%98%EC%97%AC-Async-%EC%82%AC%EC%9A%A9%ED%95%98%EA%B8%B0) 관련하여 정리하고 wiki에 작성
* ["Async를 이용하여 EventListener 사용하기"](https://github.com/selab-hs/selab-blog-api/wiki/%5B%ED%9D%AC%EC%B0%AC%5D-Async%EB%A5%BC-%EC%9D%B4%EC%9A%A9%ED%95%98%EC%97%AC-EventListener-%EC%82%AC%EC%9A%A9%ED%95%98%EA%B8%B0) 관련하여 정리하고 wiki에 작성

## 이슈

**Comment by DongGeon0908** [comment]
### 작업 사항
- 무중단 배포 서비스 구축 진행

### 이슈
- RDS 접속 에러 해결
